tools/jobify/jobify.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include "stljobs.h"
+#include <algorithm>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <Windows.h>
+
+wchar_t* get_subcommand() {
+    auto start     = ::GetCommandLineW();

```

Comment:
Why didn't `stljobs.h` use `::` for WinAPI functions, but this source file does?

Reply:
Because this file is a lot older :)